src/D2L.Bmx/OktaAuthenticator.cs

```diff
@@ -130,6 +143,121 @@ private bool TryAuthenticateFromCache(
 		return true;
 	}
 
+	private async Task<IOktaAuthenticatedClient?> TryAuthenticateWithDSSOAsync(
+		string org,
+		string user,
+		IOktaClientFactory oktaClientFactory,
+		bool experimental
+	) {
+		await using IBrowser? browser = await Browser.LaunchBrowserAsync( experimental );
+		if( browser is null ) {
+			return null;
+		}
+
+		Console.WriteLine( "Attempting to automatically login using DSSO." );
```

Comment:
Maybe I should replace all the places I use `DSSO` with `passwordless` since people probably won't know what DSSO means

Reply:
For any of the println messages

Reply:
yeah
https://github.com/Brightspace/bmx/pull/480#discussion_r1765459191